packages/cli/src/index.ts

```diff
@@ -764,6 +777,66 @@ yargs
       commandWrapper(handlerBuildCommand)(argv as Arguments<BuildDocsArgv>);
     }
   )
+  .command(
+    'translations',
+    'Creates or updates translations.yaml files and fills them with missing built-in translations and translations from the redocly.yaml and sidebars.yaml files.',
+    (yargs) =>
+      yargs.options({
+        contentDir: {
+          alias: 'd',
+          type: 'string',
+          description: 'Project content directory.',
+          default: '.',
+        },
+        locale: {
```

Comment:
If the locale is required, then let's make it a positional argument. This approach isn't completely consistent throughout the CLI but we get complaints about the main other place where we didn't get this correct (the outDir on split)

Reply:
I realise this means we can only translate one locale or all of them, but I think people will manage to repeat the command if they need to.

Reply:
I don't see any benefits of making it positional in Redocly CLI while it's an option in Realm. Wouldn't it be better to keep it as much consistent as we can?

Reply:
Redocly CLI should be the only interface to all these commands, so the consistency should be with the rest of this tool.